pkg/cmd/cli/cmd/importimage.go

```diff
@@ -243,6 +244,10 @@ func RunImportImage(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, arg
 		}
 
 		fmt.Fprintln(out, info)
+
+		if r := result.Status.Repository; r != nil && len(r.AdditionalTags) > 0 {
+			fmt.Fprintf(out, "\ninfo: The remote repository contained %d additional tags which were not imported: %s\n", len(r.AdditionalTags), strings.Join(r.AdditionalTags, ", "))
```

Comment:
"For performance" is the best we can do.  When I come back to cleanup
the output into columns I'll add that